soaks/common/terraform/modules/monitoring/observer.yaml.tpl

```diff
@@ -3,7 +3,7 @@ experiment_name: "${experiment_name}"
 variant: "${experiment_variant}"
 vector_id: "${vector_id}"
 queries:
-  - query: sum(rate((bytes_written[2s])))
+  - query: sum(rate((bytes_written[30s])))
```

Comment:
Flagging this to make sure we don't merge this up.

Reply:
Reverted in a5aa76a